### ISSUE TITLE
Corber Removing Too Much From Android Manifest.xml

### ISSUE THREAD

**Issue by MatthewPringle** [isleofcode/corber#654]
I have this in my config.xml

```
<edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
    <application android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round"/>
    <application android:usesCleartextTraffic="true"/>
    <application android:allowBackup="false"/>
</edit-config>
```

Corber seems to be removing the whole block, I believe you should only be removing `<application android:usesCleartextTraffic="true"/>`

**Comment by MatthewPringle** [comment]
This seems to happen in clean up after using Corber start and using CTRL C to end the process.

**Comment by alexblom** [comment]
Fixed as of 1.4.3. Thank you!
